canonical/ubuntu.com
# /training copy update

## Done

- Updated copy as per [doc](https://docs.google.com/document/d/1gyxy75qXC5Ta08o6Vnm13yK6aeN4qOnsETB2gZ4e0ZY/edit#heading=h.hartsr26sbq8)
- Ran djlint on target file

## QA

- View the site locally in your web browser at: https://ubuntu-com-13773.demos.haus/training
- See that the requested changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10508
